azure/ConsiderationsForServiceDesign.md

```diff
@@ -141,94 +142,201 @@ cannot collide with a resource path that contains user-specified resource ids.
 Long-running operations are an API design pattern that should be used when the processing of
 an operation may take a significant amount of time -- longer than a client will want to block
 waiting for the result.
-Azure allows for two forms of this design pattern: resource-based long-running operations (RELO),
-which is the preferred pattern, and long-running operations with a status monitor.
 
-In both patterns, the processing of the operation is initiated by one API call and the client
-obtains the results of the operation from a subsequent API call.
-Here we illustrate the sequence of API calls involved in each of these patterns.
+The request that initiates a long-running operation generally returns a `202 Accepted` response with a
+response body that contains a representation of a _status monitor_, which is an ephemeral resource that
```

Comment:
The 202 + status monitor response body should only be returned for a POST operation. So I'm not sure if the "generally" vs. "special case" terminology is the right choice.

I think we are also falling into the same trap that the old guidance did in that in order to correctly understand the guidance, you have to understand HTTP terminology. While it is true that the status monitor is a resource (in HTTP terminology), my question is how likely it is that our target audience will correctly interpret, or if their understanding of a "resource" is different. I've heard "it's not a resource, it is just a URL or location" plenty of times in my discussions with service teams.

Reply:
Now called "POST or DELETE LRO".